src/com/superiornetworks/icarus/commands/Command_report.java

```diff
@@ -72,6 +72,10 @@ public boolean run(CommandSender sender, Command cmnd, String string, String[] a
             {
                 admin.sendMessage(ChatUtils.colorize("&8[&4ICarusMod&8] &a" + sender.getName() + " &4Has reported " + Reported + " - " + player.getAddress().getAddress().getHostAddress() + " &4with the reason &2" + report_reason + "&4."));
             }
+            if (ICM_Rank.isRankOrHigher(player, ICM_Rank.Rank.SUPER))
+            {
+                sender.sendMessage(ChatColor.PURPLE + "Please do not try to report a staff member, If the staff member is abusing make sure to report him to a high enough rank to take action!");
```

Comment:
Maybe "Please do not report staff members. If a staff member is abusing, take it to the forums."

It sounds better, if you know what I mean.